pkg/karmadactl/cmdinit/utils/util_test.go

```diff
@@ -24,3 +25,24 @@ func TestDownloadFile(t *testing.T) {
 		panic(err)
 	}
 }
+
+func TestListFiles(t *testing.T) {
+	tests := []struct {
+		name string
+		path string
+		want []string
+	}{
```

Comment:
What happens to input a path that does not exist?

Reply:
Empty path will panic

Reply:
Does that have to be taken into account in the method?